### PR TITLE
feat: Checkbox + Toggle revamp

### LINKED ISSUE #8545: Checkbox and Toggle component 2.0
### Checkbox:
PRD [here](https://app.clickup.com/37484951/v/dc/13qycq-2704/13qycq-15696)
Design [here](https://www.figma.com/file/mnyECAMd6Wc7qRIDyYmYfI/Components-redesign-01?type=design&node-id=1750-566606&mode=design&t=gUdgnTdSTeyvC9J0-4)

### Toggle:
PRD [here](https://app.clickup.com/37484951/v/dc/13qycq-2704/13qycq-15296)
Design [here](https://www.figma.com/file/mnyECAMd6Wc7qRIDyYmYfI/Components-redesign-01?type=design&node-id=6634-44015&mode=design&t=gUdgnTdSTeyvC9J0-4)

### PR DESCRIPTION
**Pull request by stepinfwd**
closes #8545

### CODE REVIEW

**Review by sherfin94 — CHANGES_REQUESTED** [review]
Hey @stepinfwd , the `onChange` event is not available in the checkbox. Also, for the toggle switch, the two of the styles say 'Checked color' and 'Unchecked color', this should be something like 'color when turned on', 'color when turned off', speak to PM on this.

**Review by sherfin94 — CHANGES_REQUESTED** [review]
Hey @stepinfwd , the `onChange` event does not seem to working when I tried checking and unchecking.

**Review by sherfin94 — CHANGES_REQUESTED** [review]
Hey @stepinfwd ,

I set the `onChange` to show the value of the checkbox, but it is giving showing `false` when it should be `true` and giving `true` when it should be `false`.

image


image


image

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@stepinfwd Please resolve the conflicts and check with the design. There might be some changes

File: frontend/assets/images/icons/widgets/index.jsx [hunk 1]
```diff
@@ -53,6 +53,8 @@ import Textinput from './textinput.jsx';
 import Timeline from './timeline.jsx';
 import Timer from './timer.jsx';
 import Toggleswitch from './toggleswitch.jsx';
+import ToggleSwitchNew from './toggleswitchV2.jsx';
```
Comment (johnsoncherian) [inline frontend/assets/images/icons/widgets/index.jsx:56]: Lets Keep filename and component name same wherever possible

**Review by johnsoncherian — APPROVED** [review]

**Pull request merged**